src/autorest-core/package.json

```diff
@@ -43,6 +43,8 @@
     "jsonpath": "^0.2.11",
     "linq-es2015": "^2.4.13",
     "oav": "0.4.1",
+    "autorest-interactive": "git+https://github.com/olydis/autorest-interactive.git",
+    "azure-openapi-validator": "git+https://github.com/olydis/azure-openapi-validator.git#b45423f4560585e64f574bbffbb5cdf894cfbd53",
```

Comment:
for `azure-openapi-validator` I'd say yes, the other one I will indeed remove until Garrett has acquisition figured out.